python/src/intracktive/createHash.py

```diff
@@ -0,0 +1,43 @@
+import json
+import urllib.parse
+
+HASH_KEY = "viewerState"
+
+
+def generate_viewer_state_hash(data_url: str):
+    """
+    Generate a hash string that can be appended to a URL to load inTRACKtive with a specific viewer state.
+
+    Parameters
+    ----------
+    data_url : str
+        The URL to the data file to load in inTRACKtive.
```

Comment:
I think it would be nice to finish the documentation with the return and its type.